Sources/Classes/RSMoEngageDestination.swift

```diff
@@ -157,11 +185,11 @@ extension RSMoEngageDestination {
             // Verify if the value is of type Date or not
             // Track UserAttribute using Epoch value. Refer here: https://developers.moengage.com/hc/en-us/articles/4403905883796-Tracking-User-Attributes
             if let value = value as? String, let convertedDate = dateFrom(isoDateString: value) {
-                MoEngage.sharedInstance().setUserAttributeTimestamp(convertedDate.timeIntervalSince1970, forKey: key)
+                MoEngageSDKAnalytics.sharedInstance.setUserAttribute(convertedDate.timeIntervalSince1970, withAttributeName: key)
```

Comment:
Use existing method to convert the date to timestamp or create one & use it.

Reply:
done